CLIENT-SPECIFICATION.md

```diff
@@ -1,15 +1,15 @@
 # tldr-pages client specification
 
-**Current Specification Version:** 1.5
+**Current Specification Version:** 2.0
 
-This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client. For a list of previous versions of the specification, see the [changelog section](#Changelog) below.
+This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client. For the list of previous versions of this specification, see the [changelog section](#Changelog) below.
```

Comment:
IMO there's not much benefit to this change. Both versions strike me as correct, and I do have a slight preference for the first one. If you don't feel strongly about this change, my recommendation would be to revert it.

Reply:
```suggestion
This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client. For a list of previous versions of the specification, see the [changelog section](#Changelog) below.
```